bokehjs/src/coffee/common/plot.coffee

```diff
@@ -62,6 +74,11 @@ class PlotView extends ContinuumView
 
     @canvas_view.render()
 
+    # If requested, try enabling webgl
+    if @mget('webgl') or window.location.search.indexOf('webgl=1') > 0
```

Comment:
> I really like this webgl=1 support in the url... nice idea...

It would be an interesting idea when submitted as an idea and then implemented universally, so that I could simply `@mget(...)` and get this behavior. Currently it's a hack on top of a well established mechanism. Also, why use `0` and `1` for boolean values? I presume to save keystrokes, but our type system disagrees with this choice.